src/main/java/com/couchbase/lite/replicator/PusherInternal.java

```diff
@@ -209,23 +271,25 @@ public void beginReplicating() {
         }
         ChangesOptions options = new ChangesOptions();
         options.setIncludeConflicts(true);
-        Log.d(Log.TAG_SYNC, "%s: Getting changes since %d", this, lastSequenceLong);
+        Log.d(TAG, "%s: Getting changes since %d", this, lastSequenceLong);
         final RevisionList changes = db.changesSince(lastSequenceLong, options, filter, filterParams);
         if (changes.size() > 0) {
-            Log.d(Log.TAG_SYNC, "%s: Queuing %d changes since %d", this, changes.size(), lastSequenceLong);
+            Log.d(TAG, "%s: Queuing %d changes since %d", this, changes.size(), lastSequenceLong);
             // NOTE: Needs to submit changes into inbox from RemoteRequest thread for beginReplication.
             //       RemoteRequest thread is observed by pendingFuture, if using other thread to
             //       submit changes  into inbox, there are chance both inbox and pendingFutures are
             //       empty.
             submitRevisions(changes);
         } else {
-            Log.d(Log.TAG_SYNC, "%s: No changes since %d", this, lastSequenceLong);
+            Log.d(TAG, "%s: No changes since %d", this, lastSequenceLong);
         }
 
         // process queued changes by `changed()` callback
         synchronized (changesLock) {
```

Comment:
I think synchronize here doesn't need anymore as the code will be run in the support work executor.